plugins/modules/ecs_taskdefinition.py

```diff
@@ -46,9 +46,425 @@
+                        required: False
+                        type: str
+                        default: tcp
+                        choices: ['tcp', 'udp']
+            essential:
+                description:
+                    - If C(essential) is True, and the fails or stops for any reason, all other containers that are part of the task are stopped.
+                required: False
+                type: bool
+            entryPoint:
+                description: The entry point that is passed to the container.
+                required: False
+                type: str
+            command:
+                description: The command that is passed to the container.
+                required: False
+                type: list
+            environment:
+                description: The environment variables to pass to a container.
+                required: False
+                type: list
+                elements: dict
+                suboptions:
+                    name:
+                        description: The name of the key-value pair.
+                        required: False
+                        type: str
+                    value:
+                        description: The value of the key-value pair.
+                        required: False
+                        type: str
+            environmentFiles:
+                description: A list of files containing the environment variables to pass to a container.
+                required: False
+                type: list
+                elements: dict
+                suboptions:
+                    value:
+                        description: The Amazon Resource Name (ARN) of the Amazon S3 object containing the environment variable file.
+                        required: False
+                        type: str
+                    type:
+                        description: The file type to use. The only supported value is s3.
+                        required: False
+                        type: str
+            mountPoints:
+                description: The mount points for data volumes in your container.
+                required: False
+                type: list
+                elements: dict
+                suboptions:
+                    sourceVolume:
+                        description: The name of the volume to mount.
+                        required: False
+                        type: str
+                    containerPath:
+                        description: The path on the container to mount the host volume at.
+                        required: False
+                        type: str
+                    readOnly:
+                        description:
+                            - If this value is True, the container has read-only access to the volume.
+                            - If this value is False, then the container can write to the volume.
+                            - The default value is False.
```

Comment:
```suggestion
                            - If this value is C(True), the container has read-only access to the volume.
                            - If this value is C(False), then the container can write to the volume.
```
default is defined below, no need to duplicate

Reply:
Removed redundant phrase, thanks!